libraries/botbuilder-ai-orchestrator/src/orchestratorAdaptiveRecognizer.ts

```diff
@@ -253,6 +291,67 @@ export class OrchestratorAdaptiveRecognizer
         return recognizerResult;
     }
 
+    private getTopTwoIntents(
+        result: RecognizerResult
+    ): { name: string; score: number }[] {
+        if (!result || !result.intents) {
+            throw new Error('result is empty');
+        }
+        const intents = Object.entries(result.intents)
+                            .map((intent) => {return {name: intent[0], score: +intent[1].score} })
+                            .sort((a,b) => (b.score - a.score))
+        intents.length = 2;
```

Comment:
Why are we mutating array length directly? Should perhaps use `.slice(0, 2)` instead?